psl-core/src/test/java/org/linqs/psl/application/inference/mpe/SGDInferenceTest.java

```diff
@@ -40,7 +42,39 @@ protected InferenceApplication getInference(List<Rule> rules, Database db) {
 
     @Override
     public void initialValueTest() {
+        // SGD Non-coordinate step.
         Options.SGD_LEARNING_RATE.set(10.0);
+        Options.SGD_COORDINATE_STEP.set(false);
+        super.initialValueTest();
+
+        // SGD Coordinate step.
+        Options.SGD_COORDINATE_STEP.set(true);
+        super.initialValueTest();
+
+        cleanup();
+
+        // Adam.
+        Options.SGD_EXTENSION.set("ADAM");
```

Comment:
When possible, try to refer to the enum directly rather than the string value.
This helps ensure that the depended code doesn't break if we change the enum (since it will just fail at compile time).

See:
https://github.com/linqs/psl/blob/develop/psl-core/src/test/java/org/linqs/psl/application/learning/weight/search/grid/GridSearchContinuousLossMSETest.java#L39